src/components/tuiter/index.js

```diff
@@ -44,6 +45,7 @@ function Tuiter () {
               <Route path="/movies/:imdbID" element={<MovieDetails/>}/>
               <Route path="/restaurant/:rid" element={<BusinessHome/>}/>
               <Route path="/more" element={<More/>}/>
+              <Route path="/restaurant/:rid/edit" element={<EditRestaurant/>}/>
```

Comment:
Just noticed a discrepancy as I was reviewing Yutong's Integration1-T6F2-54 pull request - the edit button for the business profile is linked to path /profile/business/:rid/edit-restaurant, whereas this path is /restaurant/:rid/edit. Let's agree on a path before merging

Reply:
Since the business user accesses the editing feature through the profile, not the business home page, I think the path should probably start with the profile. How about we all change it to "/profile/restaurant/:rid/edit"?

Reply:
Yes, I agree - I think that path would make the most sense. In that case, @nala-he do you also want to change your profile path to /profile/restaurant/:rid rather than /profile/business/:rid for more consistency?

Reply:
No problem. If @t1yena also agrees, I will change my profile path to /profile/restaurant/:rid and update my edit link to /profile/restaurant/:rid/edit.

Reply:
Actually, according to our conversation in #7, Macee noticed that at path /profile/restaurant/:rid and /profile/restaurant/:rid/edit, both the Profile and Restaurant are highlighted in the navbar. Since we are using both profile and restaurant keywords in the path for the nav bar button and it highlighted both. In this case, can we all use the path name /profile/business/:rid to keep it easy?

Reply:
I changed the path name to  /profile/business/:rid